Gemfile.lock

```diff
@@ -1,15 +1,15 @@
 PATH
   remote: .
   specs:
-    pharos-cluster (1.3.0.rc.1)
+    pharos-cluster (1.3.0.rc.3)
```

Comment:
What do you mean, exactly?

The gemspec contains `k8s-client ~> 0.3.3` because that version contains fixes/changes that `pharos-cluster` relies on, but the `Gemfile.lock` contains k8s-client 0.3.4 because that's the newest matching version.

Should I also update the gemspec to use `0.3.4`?